Util/MediaTypeNegotiatorInterface.php

```diff
@@ -11,16 +11,9 @@
 
 namespace FOS\RestBundle\Util;
 
-use Symfony\Component\HttpFoundation\Request;
-
+/**
+ * @deprecated since version 1.7, to be removed in 2.0. Use {@link Negotiation\FormatNegotiatorInterface} instead.
```

Comment:
since this is going into master the deprecation notices should go into the 1.7 branch